bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -55,25 +55,25 @@ channel-group-type.netatmo.noise.label = Noise
 channel-group-type.netatmo.person.label = Person
 channel-group-type.netatmo.person.channel.last-seen.label = Last Seen
 channel-group-type.netatmo.person.channel.last-seen.description = Moment when this person was last seen.
-channel-group-type.netatmo.plug.label = Thermostat Plug
 channel-group-type.netatmo.presence.label = Presence Camera
 channel-group-type.netatmo.pressure-extended.label = Pressure
 channel-group-type.netatmo.pressure-extended.channel.trend.label = Pressure Trend
 channel-group-type.netatmo.pressure.label = Pressure
+channel-group-type.netatmo.properties-room.label = Room Status
+channel-group-type.netatmo.properties-thermostat.label = Thermostat
 channel-group-type.netatmo.rain.label = Rain gauge
 channel-group-type.netatmo.rain.channel.sum-1.label = Rain 1h
 channel-group-type.netatmo.rain.channel.sum-1.description = Quantity of water over last hour.
 channel-group-type.netatmo.rain.channel.sum-24.label = Rain 24h
 channel-group-type.netatmo.rain.channel.sum-24.description = Quantity of water during the current day.
-channel-group-type.netatmo.properties-room.label = Room Status
-channel-group-type.netatmo.temperature-room.label = Room Temperature
 channel-group-type.netatmo.security.label = Home Security
 channel-group-type.netatmo.setpoint.label = Setpoint
 channel-group-type.netatmo.setpoint.channel.end.label = Setpoint End
 channel-group-type.netatmo.setpoint.channel.end.description = End time of the currently applied setpoint.
 channel-group-type.netatmo.setpoint.channel.start.label = Setpoint Start
 channel-group-type.netatmo.setpoint.channel.start.description = Start time of the currently applied setpoint.
 channel-group-type.netatmo.signal.label = Signal
+channel-group-type.netatmo.siren.label = Siren Status
```

Comment:
Some entries are missing for the two channels of this new group.

Reply:
Ok, adding them